src/server.js

```diff
@@ -70,6 +70,15 @@ module.exports = function(options) {
 	}
 };
 
+function reqHeader(req, header, defValue) {
+	var headerValue = req.get(header)
+	if (headerValue) {
+		return headerValue
+	} else {
+		return defValue
+	}
+}
+
```

Comment:
I think that helper is unnecessary, you can handle the whole thing with a 3rd dict argument to [this `_.merge()`](https://github.com/lucamilanesio/shout/blob/get-default-user-identity-from-header/src/server.js#L85)